doc/source/whatsnew/v0.19.0.txt

```diff
@@ -436,6 +438,15 @@ Deprecations
 - top-level ``pd.ordered_merge()`` has been renamed to ``pd.merge_ordered()`` and the original name will be removed in a future version (:issue:`13358`)
 - ``Timestamp.offset`` property (and named arg in the constructor), has been deprecated in favor of ``freq`` (:issue:`12160`)
 
+.. _whatsnew_0190.deprecations.read_csv_arg_rename:
+
+All multi-word arguments to ``pd.read_csv()`` will now have their individual words separated by an underscore, and their
+ underscore-less versions have not be deprecated (:issue:`13386`). They are:
+
+- The ``chunksize``, ``dayfirst``, ``doublequote``, ``escapechar``, ``lineterminator``, ``quotechar``, ``skipinitialspace``,
```

Comment:
@jreback : GitHub unfortunately hid away the other comment about the table, so I'm "refreshing it."  Could you explain how to create tables in these files?